ctn_waterloo/content/publications/morcos2018.bib

```diff
@@ -0,0 +1,38 @@
+@inproceedings{morcos2018,
+    author      = {Morcos, Benjamin and Stewart, Terrence C and
+                   Eliasmith, Chris and Kapre, Nachiket},
+    title       = {Implementing NEF Neural Networks on Embedded FPGAs},
+    booktitle   = {International Conference on Field-Programmable
+                   Technology (FPT)},
+    year        = {2018},
+    eventdate   = {2018-12-11/2018-12-14},
+    location    = {Naha, Okinawa, Japan},
+    publisher   = {IEEE},
+    pages       = {25-32},
```

Comment:
Similarly where did you get this page range from?

Reply:
Similarly, page range is from the copy of the proceedings I received at the conference.